stream/src/main/scala/org/apache/pekko/stream/scaladsl/Hub.scala

```diff
@@ -476,7 +476,8 @@ object BroadcastHub {
 /**
  * INTERNAL API
  */
-private[pekko] class BroadcastHub[T](startAfterNrOfConsumers: Int, bufferSize: Int)
+private[pekko] class BroadcastHub[T](startAfterNrOfConsumers: Int, bufferSize: Int,
+    registrationPendingCallback: Long => Unit = _ => ())
```

Comment:
Add a doc about the `registrationPendingCallback` 's function input and expected output?

Reply:
Added both in [4596113](https://github.com/apache/pekko/pull/1841/commits/4596113cd124f72f3000bd2bd899605506089bd9)